admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -2,35 +2,43 @@
 Encryption Configuration
 ========================
 
-ownCloud includes a server-side encryption application. The Encryption app 
-encrypts all files stored on the ownCloud server, and all files on remote 
-storage that is connected to your ownCloud server. Encryption and decryption are 
-performed on the ownCloud server. All files sent to remote storage (for example 
-Dropbox and Google Drive) will be encrypted by the ownCloud server, and upon 
-retrieval, decrypted before serving them to you and anyone you have shared them 
-with.
-
-.. note:: Encrypting files increases their size by roughly 35%, so you must take 
-   this into account when you are provisioning storage and setting storage 
-   quotas. User's quotas are based on the unencrypted file size, and not the 
-   encrypted file size.
+In ownCloud 8.1 the Encryption app has been changed to the ownCloud Default 
+Encryption Module. It has a number of changes and improvements, including:
+
+* When encryption is enabled, all files are no longer encrypted at user's first 
+  logins because this caused timeouts on large installations. Instead, only 
+  files that are created or updated after encryption has been enabled are 
+  encrypted.
+
+* The  "decrypt all" option in the Personal settings has been removed, also for 
+  performance reasons.
+
+* A new option for users to enable/disable encryption on a per mount-point 
+  basis.
+
+The Default Encryption Module encrypts files stored on the ownCloud server, and 
+files on remote storage that is connected to your ownCloud server. Encryption 
+and decryption are performed on the ownCloud server. All files sent to remote 
+storage (for example Dropbox and Google Drive) will be encrypted by the ownCloud 
+server, and upon retrieval, decrypted before serving them to you and anyone you 
+have shared them with.
+
+.. note:: Encrypting files increases their size by roughly 35%, so you must 
+   take this into account when you are provisioning storage and setting 
+   storage quotas. User's quotas are based on the unencrypted file size, and 
+   not the encrypted file size.
 
 When files on external storage are encrypted in ownCloud, you cannot share them 
 directly from the external storage services, but only through ownCloud sharing 
 because the key to decrypt the data never leaves the ownCloud server.
 
-The main purpose of the Encryption app is to protect users' files on remote 
-storage, and to do it easily and seamlessly from within ownCloud. 
-
-The Encryption app generates a strong encryption key, which is unlocked by 
-user's passwords. So your users don't need to track an extra password, but 
-simply log in as they normally do.
-
-Encryption is applied server-wide; it cannot be applied to selected users or 
-files.
+The main purpose of the Default Encryption Module is to protect users' files on 
```

Comment:
ownCloud encryption now consists of two parts. The underlying ownCloud encryption system, this is what you can enable/disable in the admin settings and the encryption module (which is a app, at the moment the only module we provide is the "Default Encryption Module"). In order to use encryption you need both, enable the ownCloud encryption system in the admin settings and load at least one encryption module (app).

So I would rephrase the sentence in line 35: "The main porpose of the ownCloud server-side-encryption is to protect users' files on..." Because this will be the same for every encryption module we might develop in the future.